ci/provisioning/post_provision_config_common_functions.sh

```diff
@@ -413,34 +433,35 @@ post_provision_config_nodes() {
         if ! retry_dnf 360 install "${inst_rpms[@]/%/${DAOS_VERSION:-}}"; then
             rc=${PIPESTATUS[0]}
             dump_repos
+            echo "Failed to install packages"
             return "$rc"
         fi
     fi
 
     if lspci | grep "ConnectX-6" && ! grep MOFED_VERSION /etc/do-release; then
-        # Need this module file
-        version="$(rpm -q --qf "%{version}" openmpi)"
-        mkdir -p /etc/modulefiles/mpi/
-        cat << EOF > /etc/modulefiles/mpi/mlnx_openmpi-x86_64
-#%Module 1.0
-#
-#  OpenMPI module for use with 'environment-modules' package:
-#
-conflict		mpi
-prepend-path 		PATH 		/usr/mpi/gcc/openmpi-$version/bin
-prepend-path 		LD_LIBRARY_PATH /usr/mpi/gcc/openmpi-$version/lib64
-prepend-path 		PKG_CONFIG_PATH	/usr/mpi/gcc/openmpi-$version/lib64/pkgconfig
-prepend-path		MANPATH		/usr/mpi/gcc/openmpi-$version/share/man
-setenv 			MPI_BIN		/usr/mpi/gcc/openmpi-$version/bin
-setenv			MPI_SYSCONFIG	/usr/mpi/gcc/openmpi-$version/etc
-setenv			MPI_FORTRAN_MOD_DIR	/usr/mpi/gcc/openmpi-$version/lib64
-setenv			MPI_INCLUDE	/usr/mpi/gcc/openmpi-$version/include
-setenv	 		MPI_LIB		/usr/mpi/gcc/openmpi-$version/lib64
-setenv			MPI_MAN			/usr/mpi/gcc/openmpi-$version/share/man
-setenv			MPI_COMPILER	openmpi-x86_64
-setenv			MPI_SUFFIX	_openmpi
-setenv	 		MPI_HOME	/usr/mpi/gcc/openmpi-$version
-EOF
+#         # Need this module file
+#         version="$(rpm -q --qf "%{version}" openmpi)"
+#         mkdir -p /etc/modulefiles/mpi/
+#         cat << EOF > /etc/modulefiles/mpi/mlnx_openmpi-x86_64
+# #%Module 1.0
+# #
+# #  OpenMPI module for use with 'environment-modules' package:
+# #
+# conflict		mpi
+# prepend-path 		PATH 		/usr/mpi/gcc/openmpi-$version/bin
+# prepend-path 		LD_LIBRARY_PATH /usr/mpi/gcc/openmpi-$version/lib64
+# prepend-path 		PKG_CONFIG_PATH	/usr/mpi/gcc/openmpi-$version/lib64/pkgconfig
+# prepend-path		MANPATH		/usr/mpi/gcc/openmpi-$version/share/man
+# setenv 			MPI_BIN		/usr/mpi/gcc/openmpi-$version/bin
+# setenv			MPI_SYSCONFIG	/usr/mpi/gcc/openmpi-$version/etc
+# setenv			MPI_FORTRAN_MOD_DIR	/usr/mpi/gcc/openmpi-$version/lib64
+# setenv			MPI_INCLUDE	/usr/mpi/gcc/openmpi-$version/include
+# setenv	 		MPI_LIB		/usr/mpi/gcc/openmpi-$version/lib64
+# setenv			MPI_MAN			/usr/mpi/gcc/openmpi-$version/share/man
+# setenv			MPI_COMPILER	openmpi-x86_64
+# setenv			MPI_SUFFIX	_openmpi
+# setenv	 		MPI_HOME	/usr/mpi/gcc/openmpi-$version
+# EOF
```

Comment:
I think we should just remove this

Reply:
Removed in https://github.com/daos-stack/daos/pull/16224/commits/0172aa3193ecd77fa6c2788a2a1bf92820cb5138